src/calculateRentalCost.js

```diff
@@ -2,9 +2,30 @@
  * @param {number} days
  *
  * @return {number}
+ *
+ *
+ *
  */
+
+const PRICE = 40;
+
+const DAYS_SMALL_DISCOUNT = 3;
+const SMALL_DISCOUNT = 20;
+
+const DAYZ_BIG_DISCOUNT = 7;
+const BIG_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
   // write code here
+  if (days >= DAYS_SMALL_DISCOUNT && days < DAYZ_BIG_DISCOUNT) {
+    return days * PRICE - SMALL_DISCOUNT;
+  }
```

Comment:
You can don't use this hard construction if you first will to compare  DAYZ_BIG_DISCOUNT with day. For example

if (days >= DAYZ_BIG_DISCOUNT) {
    return days * PRICE - BIG_DISCOUNT;
  }

if (days >= DAYS_SMALL_DISCOUNT) {
    return days * PRICE - SMALL_DISCOUNT;
  }